### ISSUE TITLE
Dir does not change in Ajax Sorting - Always sends dir=asc

### ISSUE THREAD

**Issue by adityawritso** [olifolkerd/tabulator#720]
Hi There,

Ajax sorting is not working properly for me. When I click for the first time then it sends {column: Id, dir: asc}, when I click again on the same column just after ideally it should send {column: Id, dir: desc} but it is again sending the same object {column: Id, dir: asc}.

Below, is the code snippet which I am using 
```
var tabulatorData = [
	{title: 'Id', field: 'Id', headerSort:true},
	{title: 'Name', field: 'Name', headerSort:true},
	{title: 'Age', field: 'Age', headerSort:true},
];

$("#user-table").tabulator({
    pagination:"remote",
    paginationSize : 50,
    layout:"fitDataFill",                            
    placeholder:"No Data Set",
    height:"400px",
    columns: tabulatorData,
    ajaxURL:"user-records", 
    ajaxSorting:true,
    ajaxParams:{index: 'active'}
   
});

```
Am I missing something?

Please, help.

**Comment by olifolkerd** [comment]
Hey @adityawritso,

This has already been identified as a bug, there should be a fix comming out for it in this weeks patch release.

Thanks for getting in touch

Cheers

Oli

**Comment by TomBruyneel** [comment]
Hey @olifolkerd has this patch release already been released? I don't see any commits in the git?

Thanks for all the support!

**Comment by adityawritso** [comment]
Hey @olifolkerd, Please let me know when the patch for this bug will release.

Thanks for all the support!

**Comment by olifolkerd** [comment]
Hey @adityawritso I have just being trying to fix this but cant seem to replicate the issue, do you have a table constructor example and a description of exactly what you press to trigger the issue.

Cheers

Oli

**Comment by olifolkerd** [comment]
looking into it further, this only appears to be an issue when a malformed ajax request is received, could you confirm there are no warnings about your ajax request in the logs?

**Comment by olifolkerd** [comment]
I think I have gotten to the bottom of this, I pushed a fix to it to master back in October but completely forgot to do an official release for it, I have just released patch fix 3.3.2 which includes the fix for this,

Let me know if you are still having issues after the update.

Cheers

Oli

**Comment by adityawritso** [comment]
Hay @olifolkerd 

Thanks a lot. It is working perfectly now. :+1:
